pkg/commands/internal/migrate/migrate_linter_names.go

```diff
@@ -0,0 +1,922 @@
+package migrate
+
+import (
+	"cmp"
+	"slices"
+
+	"github.com/golangci/golangci-lint/pkg/commands/internal/migrate/ptr"
+	"github.com/golangci/golangci-lint/pkg/commands/internal/migrate/versionone"
+)
+
+type LinterInfo struct {
+	Name             string   `json:"name"`
+	Presets          []string `json:"inPresets"`
+	Slow             bool     `json:"isSlow,omitempty"`
+	Default          bool     `json:"enabledByDefault,omitempty"`
+	AlternativeNames []string `json:"alternativeNames,omitempty"`
+}
+
+func (l *LinterInfo) isName(name string) bool {
+	if name == l.Name {
+		return true
+	}
+
+	return slices.Contains(l.AlternativeNames, name)
+}
+
+func (l *LinterInfo) hasPresets(names []string) bool {
+	for _, name := range names {
+		if slices.Contains(l.Presets, name) {
+			return true
+		}
+	}
+
+	return false
+}
+
+func ProcessEffectiveLinters(old versionone.Linters) (enable, disable []string) {
+	switch {
+	case ptr.Deref(old.DisableAll):
+		return disableAllFilter(old), nil
+	case ptr.Deref(old.EnableAll):
+		return nil, enableAllFilter(old)
```

Comment:
These lines confuse me and can confuse future readers. I propose to extend function names with `enable` and `disable` words for more clarity.

E.g.,
```suggestion
	case ptr.Deref(old.DisableAll):
		return enableWhenDisableAllFilter(old), nil
	case ptr.Deref(old.EnableAll):
		return nil, disableLintersWhenEnableAllFilter(old)
```

Reply:
I don't agree